### PR TITLE
New mail capture form to send messages to Segment.io

### LINKED ISSUE #434: Switch from Mixpanel to Customer.io/Segment for email sign ups

### PR DESCRIPTION
**Pull request by pitchinvasion**
Closes #434

    Note that this is just for the library so we can propogate it to other
    sites, the actual switch to Segment will be addressed by #439

### CODE REVIEW

File: source/javascripts/email_capture.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      });
+
+      function submitValues() {
+        var values = {
+          email: email.val(),
+          newsletter: true,
+        };
+        var segments = email.data("segments");
+
+        if (segments) {
```
Comment (spike01) [inline source/javascripts/email_capture.js]: Maybe wrap this in an object? :grin:

**Comment by spike01** [comment]
Too much whitespace :shipit:

File: source/javascripts/email_capture.js (hunk 1 above)
Comment (pitchinvasion) [inline source/javascripts/email_capture.js]: I can't tell if that's an ironic emoji or not....

Comment (spike01) [inline source/javascripts/email_capture.js]: After yesterday's discussion, yes.

Comment (pitchinvasion) [inline source/javascripts/email_capture.js]: Thought so. I'm gradually learning human interaction skills clearly

Comment (spike01) [inline source/javascripts/email_capture.js]: Maybe :grin:

File: source/javascripts/email_capture.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ - Form should have one <input> element with attribute 'type=email'
+ - Form should have one label element with attribute 'for=email_input_id' (used to display error/success message)
+ */
+
+(function ($) {
+  'use strict';
+
+  $.fn.emailCapture = function (options) {
+
+    return $(this).each(function(index, element) {
```
Comment (tansaku) [inline source/javascripts/email_capture.js]: should this be a named function?

File: source/javascripts/email_capture.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      var form = $(element);
+      var email = form.find('input[type=email]');
+      var label = form.find('label[for=' + email.attr('id') + ']');
+      var submitMsg = "Thank you for subscribing!";
+
+      form.on("submit", function(event) {
```
Comment (tansaku) [inline source/javascripts/email_capture.js]: and this one too?

**Comment by tansaku** [comment]
@pitchinvasion basically good, but wonder if it could be adjusted to make it clearer what everything does and what it's for by using more named functions etc.

File: source/javascripts/email_capture.js (hunk 3 above)
Comment (pitchinvasion) [inline source/javascripts/email_capture.js]: I'll add one for this one, the other pattern with an anonymous function/each is a standard jQuery plugin pattern so I'll leave that one in

**Pull request merged**